antarest/study/business/model/hydro_model.py

```diff
@@ -18,6 +18,9 @@
 from antarest.core.utils.string import to_camel_case
 from antarest.study.business.all_optional_meta import all_optional_model, camel_case_model
 
+INFLOW_PATH = ["input", "hydro", "prepro", "{area_id}", "prepro", "prepro"]
```

Comment:
better to juste have a `inflow_path` function which takes the area ID as argument, instead of having to perform some string replacement